infra/src/main/kotlin/fi/fta/geoviite/infra/linking/PublishService.kt

```diff
@@ -285,12 +344,15 @@ class PublishService @Autowired constructor(
         }
     }
 
+    fun validateKmPostOwnInformation(id: IntId<TrackLayoutKmPost>): List<PublishValidationError> {
```

Comment:
Näitä nimiä lukiessa en ole ihan varma mihin tuo "Own" viittaa. Oletettavasti siihen, että validoidaan ka. olion kentät, mutta ei esim. mahdollisia linkityksiä muihin paikannuspohjan käsitteisiin (tosin tuolla refelinelle haetaan kyllä tracknumber...)? Lähinnä mietin että voisiko nämä nimetä jotenkin kuvaavammin, mutta en itsekään keksi heti parempaa nimeä, kun en ole ihan varma mitä näiden kuuluu tarkalleen ottaen tehdä kullakin asset-typpillä

Reply:
Ihan hyvä pointti. Ehkä validateFields & validateLinks voisi kuvata eroa?

Reply:
Ei nyt oikein siinä muodossa, syystä että vaihtoehdothan tässä on validoida joko kaikki olioon liittyvä tieto julkaistavan joukon skoopissa, tai vain sen oma tieto. Alkuaan olin nimennyt jälkimmäiset muotoon validateFooUnit, mutta se vasta hämmentävä olikin, kun juuri toisin päin voisi ajatella, että eikös nämä Unit-päätteiset metodit ole ne, jotka liittyy julkaisun validointiin julkaistavana yksikkönä.